examples/openIMIS/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+version: "2.4"
+
+services:
+  db:
+    container_name: ${PROJECT_NAME:-openimis}-db
+    image: ghcr.io/openimis/openimis-mssql:${DB_TAG:-latest}
+    restart: always
+    environment:
+      - DB_USER_PASSWORD=${DB_PASSWORD}
+      - SA_PASSWORD=${DB_PASSWORD}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - ACCEPT_EULA=${ACCEPT_EULA}
+    healthcheck:
+      test: "bash /app/healthcheck.sh"
+      interval: 10s
+      timeout: 5s
+      retries: 5
+      start_period: 100s
+    networks:
+      openimis-net:
+    volumes:
+      - database:/var/opt/mssql/data
+    ports:
+      - 1433:1433
+  backend:
+    build:
+      context: https://github.com/openimis/openimis-be_py.git#Test_Harness_Setup
+      args:
+        OPENIMIS_CONF_JSON: ${OPENIMIS_BE_CONF_JSON}
+    image: openimis/backend:${PROJECT_NAME:-local}
+    volumes:
+      - photos:/openimis-be/openIMIS/images/insurees
+      - ./import_data.py:/openimis-be/openIMIS/custom_scripts/import_data.py
+      - ./check_service_availability.sh:/openimis-be/openIMIS/custom_scripts/check_service_availability.sh
```

Comment:
Importing it to the server through volume seems a bit hacky. In the long term we could think about custom backend Dockerfile used exclusively by GovStack test harness, but for the time being we can keep it in current form. If we'll need to have more contributions from example/openIMIS to container, we will need to switch for Dockerfile.